OpenRA.Mods.Common/Traits/BotModules/SquadManagerBotModule.cs

```diff
@@ -46,6 +47,9 @@ public class SquadManagerBotModuleInfo : ConditionalTraitInfo
 		[Desc("Own actor types that are prioritized when defending.")]
 		public readonly HashSet<string> ProtectionTypes = new();
 
+		[Desc("Target types are used for identifying aircraft.")]
+		public readonly BitSet<TargetableType> AircraftTargetType = new("Air", "AirborneActor");
```

Comment:
this should have just `Air` type. The `AirborneActor` overwrite should only be applied to RA

Reply:
Fixed